mcs/class/corlib/Test/System.Threading.Tasks/TaskFactoryTest.cs

```diff
@@ -586,8 +585,9 @@ public void FromAsync_BeginCallback ()
 					Assert.IsFalse (Thread.CurrentThread.IsThreadPoolThread, "#12");
 
 					called2 = true;
-					b.Invoke (null);
-					return null;
+					var ar = Task.CompletedTask;
+					b.Invoke (ar);
```

Comment:
b.Invoke(null) throws NRE on .NET Core and .NET Framework so I fixed it this way